### DISCUSSION TITLE
Port to Eto.Forms

### DISCUSSION THREAD

**Discussion by rafntor** [Live-Charts/LiveCharts2#381] [Show and tell]
Hi, and thanks for a fine library !

I have made a port targeting [Eto.Forms](https://github.com/picoe/Eto) ;
Would you welcome a PR and bring this into your project ? (I would try to help out and assist in issues within reason)
![image](https://user-images.githubusercontent.com/44544090/151697850-33d81c90-da5c-4406-bbce-48b05bfec578.png)

**Comment by rafntor** [comment]
EtoFormsSample on Windows / WPF:
![wpfdemo](https://user-images.githubusercontent.com/44544090/151693514-4ad24a59-547a-4e2a-8553-678f6d558c94.gif)

**Comment by rafntor** [comment]
On Linux / GTK (kubuntu)
![gtkdemo](https://user-images.githubusercontent.com/44544090/151693549-7337aa21-bbf7-470d-96c7-9c8d071252bc.gif)

**Comment by rafntor** [comment]
On MacOS 
![macdemo](https://user-images.githubusercontent.com/44544090/151693577-92917ff3-bb85-4395-ac21-c64cecfdf9ed.gif)

**Comment by beto-rodriguez** [comment]
Hi @rafntor 

This looks fantastic, a PR is welcome.

Luckily the library shares around 95% of the code between all the platforms, luckily it would not require a huge amount of time to maintain it, your help is always appreciated on that.

Just let me know if I can help you in the process, your PR is welcome!

**Comment by rafntor** [comment]
Great, thanks!
I will start with a PR then and we take it from there ;-)
